RUFAS/output_manager.py

```diff
@@ -138,6 +139,13 @@ def __init__(self) -> None:
             self.saved_pool_chunks_num: int = 0
             self.saved_pool_chunks_path: Path | None = None
 
+            self.available_memory: int = 0
+            self.average_add_variable_call_addition: int | None = None
+            self.add_variable_call = 0
+            self.save_chunk_threshold_call_count: int | None = None
```

Comment:
```suggestion
            self.save_chunk_threshold_call_count: int = 0
```
Maybe it would be better to have this be 0 as the marker that a different method for deciding when to dump the pool should be used? I don't think it would make a huge difference, but I'm guessing it will lead to fewer Mypy errors being generated, and since `run_startup_sequence` takes in an int to set this attribute I think it would be reasonable to just type it as an int.